app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class BooksController < ApiController
+      def index
+        render json: Book.all
+      end
+
+      def show
+        render json: Book.find(params[:id])
```

Comment:
Accordingly to apiary, the info to return in index and show is different. You can create more than one serializer for a certain model, and you can specify which one you want to use here. For example
`render json: Book.find(params[:id]), serializer: FullBookSerializer`